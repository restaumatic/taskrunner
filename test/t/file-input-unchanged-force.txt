# no toplevel

echo lol > input.txt
git init -q
git add input.txt

go() {
  taskrunner -n mytask "$@" bash -e -c '
    snapshot input.txt
    echo "Expensive computation"
  '
}

go

echo "should compute again:"
go --force

echo "should not compute again:"
go
