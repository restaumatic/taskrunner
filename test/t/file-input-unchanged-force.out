-- output:
[mytask] info | Inputs changed, running task
[mytask] stdout | Expensive computation
[mytask] info | success
should compute again:
[mytask] info | Inputs changed, running task
[mytask] stdout | Expensive computation
[mytask] info | success
should not compute again:
